backend/pyproject.toml

```diff
@@ -10,6 +10,8 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.11"
 
+dropbox = "^12.0.2"
+itsdangerous = "^2.2.0"
```

Comment:
I don't see the use of this package 🤔 ? was this needed somewhere ?

Reply:
It is for the SessionMiddleware